torch/_functorch/partitioners.py

```diff
@@ -718,6 +697,19 @@ def classify_nodes(joint_module):
     if len(required_bw_nodes) == 0:
         return default_partition(joint_module, _joint_inputs, num_fwd_outputs=num_fwd_outputs)
 
+    def is_fusible(a, b):
+        # We can perform "memory fusion" into a cat, but cat cannot be a
+        # producer to a fusion
+        if get_aten_target(b) == aten.cat:
+            return True
```

Comment:
should this be ?
```
if get_aten_target(b) == aten.cat and get_aten_target(a): 
	return False
```

Reply:
The idea is that you can always "fuse" ops into cat, so recomputing an op that feeds into cat will always be free. But this was here before - this is just code movement.